use-cases/domain-whitelabel.md

```diff
@@ -0,0 +1,5 @@
+# How to Setup a Domain Whitelabel
+
+You can find documentation for how to setup a domain whitelabel via the UI [here](https://sendgrid.com/docs/Classroom/Basics/Whitelabel/setup_domain_whitelabel.html) and via API [here](https://github.com/sendgrid/sendgrid-ruby/blob/master/USAGE.md#whitelabel).
+
+Find more information about all of SendGrid's whitelabeling related documentation [here](https://sendgrid.com/docs/Classroom/Basics/Whitelabel/index.html).
```

Comment:
Instead of having short "here" links, which makes it harder to hit (especially on mobile devices), can you rewrite these links a little more like this? Instead of:

> You can find documentation for how to setup a domain whitelabel via the UI [here] and via API [here]

Something more like:

    You can find documentation for how to [set up a domain whitelabel via the UI], and [set up a domain whitelabel via the API].

Reply:
completed.